src/Receiver.cc

```diff
@@ -785,30 +599,27 @@ Receiver::trySendGrants()
  *      Reminder to hold the Receiver::schedulerMutex during this call.
  */
 void
-Receiver::schedule(Receiver::Message* message, const SpinLock::Lock& lock)
+Receiver::schedule(Receiver::Message* message,
+                   [[maybe_unused]] SpinLock::Lock& lock)
```

Comment:
What version of C++/gcc are you assuming? The project currently checks for C++11.